net-mail/qmailadmin/qmailadmin-1.2.15-r1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
No change in file - still updated copyright.
Don't know if important.

Reply:
Found some changes to shove in ;)